src/mpi/romio/adio/ad_lustre/ad_lustre_cyaml.h

```diff
@@ -0,0 +1,160 @@
+/*
+ * LGPL HEADER START
```

Comment:
Check with your Cray friends, but I am pretty sure our vendor partners will flip out if we try to include anything LGPL.